icu4c/source/i18n/calendar.cpp

```diff
@@ -639,7 +639,7 @@ static const int32_t kCalendarLimits[UCAL_FIELD_COUNT][4] = {
     {           0,            0,            59,            59  }, // MINUTE
     {           0,            0,            59,            59  }, // SECOND
     {           0,            0,           999,           999  }, // MILLISECOND
-    {-12*kOneHour, -12*kOneHour,   12*kOneHour,   30*kOneHour  }, // ZONE_OFFSET
+    {-16*kOneHour, -16*kOneHour,   12*kOneHour,   30*kOneHour  }, // ZONE_OFFSET
```

Comment:
Question: What is the difference between "minimum" and "greatest min"?

Reply:
[UCAL_LIMIT_MINIMUM](https://source.chromium.org/chromium/chromium/src/+/main:third_party/icu/source/i18n/unicode/calendar.h;drc=4f130855d9bc6894ecf77a8230c8071c3a2c96be;l=1614) is used to return in 
https://unicode-org.github.io/icu-docs/apidoc/dev/icu4c/classicu_1_1Calendar.html#a1a75a6ed6500cf7e36977ada0cc4412f


[UCAL_LIMIT_GREATEST_MINIMUM](https://source.chromium.org/chromium/chromium/src/+/main:third_party/icu/source/i18n/unicode/calendar.h;drc=4f130855d9bc6894ecf77a8230c8071c3a2c96be;l=1615)) is used to return in 
https://unicode-org.github.io/icu-docs/apidoc/dev/icu4c/classicu_1_1Calendar.html#a03311976955b27d0ba49d31059619fb6

Reply:
OK, a typical ICU function with little helpful documentation. :)

If the min and greatest-min were equal before, it makes sense to keep them equal now

Reply:
This also leads a question that the data for least max/greatest max. These are now +16 hours and +30 hours.

There are some Pacific islands actively using UTC+14 offset. I don't really know any positive offset beyond +14 has been ever used (or such data in tz database exists.